mods/ts/rules/defaults.yaml

```diff
@@ -609,6 +609,17 @@
 	Tooltip:
 		Name: Palette
 
+^Tunnel:
+	AlwaysVisible:
+	Immobile:
```

Comment:
I can't as buildings are heightmap aware and occupy space so they can't be placed on cliff edges:

![image](https://cloud.githubusercontent.com/assets/756669/9680715/1fc7fca2-52f9-11e5-83c0-8bae4b4f59db.png)

![image](https://cloud.githubusercontent.com/assets/756669/9680723/30b5f384-52f9-11e5-8f5f-d2cda44cd5fa.png)